test/utils/test-utils.js

```diff
@@ -80,6 +82,35 @@ function runAndGetWatchProc(testCase, args = [], setOutput = true) {
     return webpackProc;
 }
 
+const runInitWithAnswers = async (location, answers) => {
```

Comment:
Let's make this generic and not tied to init, maybe we can have a third param which takes in a runner.

Reply:
Yes we can, good idea! But instead of taking runner should take params to be passed to the CLI.
This function then could be used for e2e tests :)

Reply:
Yes, that would be better